source/nodejs/adaptivecards-designer/src/peer-command.ts

```diff
@@ -7,7 +7,10 @@ export class PeerCommand {
         let buttonElement = document.createElement("button");
         buttonElement.classList.add("acd-peerButton");
         buttonElement.type = "button";
+        buttonElement.setAttribute("role", "button");
         buttonElement.title = this.toolTip ? this.toolTip : this.name;
+        buttonElement.setAttribute("aria-label", this.toolTip ? this.toolTip : this.name);
```

Comment:
less a bug, but an opportunity to use a language feature:

```suggestion
        buttonElement.setAttribute("aria-label", this.toolTip ?? this.name);
```